<ui version="4.0" >
 <class>DumpRepoDlg</class>
 <widget class="QWidget" name="DumpRepoDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dump repo</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="1" >
      <widget class="KUrlRequester" name="m_ReposPath" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="m_RepoLabel" >
       <property name="text" >
        <string>Repository to dump:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="m_OutfileLabel" >
       <property name="text" >
        <string>Dump into:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="KUrlRequester" name="m_OutputFile" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_incrementalDump" >
     <property name="text" >
      <string>incremental Dump</string>
     </property>
     <property name="shortcut" >
      <string/>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_UseDeltas" >
     <property name="text" >
      <string>Use deltas</string>
     </property>
     <property name="shortcut" >
      <string/>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_Rangeonly" >
     <property name="text" >
      <string>Dump revision range</string>
     </property>
     <property name="shortcut" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>End revision:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Start revision:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="KIntNumInput" name="m_EndNumber" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>-1 for Head</string>
       </property>
       <property name="value" >
        <number>-1</number>
       </property>
       <property name="minValue" stdset="0" >
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KIntNumInput" name="m_StartNumber" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>-1 for Start</string>
       </property>
       <property name="value" >
        <number>-1</number>
       </property>
       <property name="minValue" stdset="0" >
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_Rangeonly</sender>
   <signal>toggled(bool)</signal>
   <receiver>DumpRepoDlg</receiver>
   <slot>slotDumpRange(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
